### PR TITLE
update Gaussian Mixture Model example with `pm.NormalMixture`

### PR DESCRIPTION
**Pull request by drbenvincent**
Following discussion in #102, this is a revamp of the GMM example. The model/notebook is now considerably simpler as it uses the new `pm.NormalMixture` distribution.

### CODE REVIEW

**Comment by drbenvincent** [comment]
<p>done</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/310/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by drbenvincent** [comment]
<p>Have updated the verbs</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/310/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by drbenvincent** [comment]
<p>I've played about with this a bit, but the plotting is not working. I'm leaning towards keeping the existing code because it is very transparent about what is happening, and use of <code>XrContinuousRV</code> would only save a couple of lines of code. </p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/310/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by drbenvincent** [comment]
Think I've addressed all the points - although let me know about `xarray-einstats`. I don't think it really it's ultra compelling to use in this simple example where the plot code is pretty concise to begin with

**Comment by OriolAbril** [comment]
I forgot the means in the xarray code, so now the plot is getting a 4d array. The difference is in how the operations are done, mean first vs mean later. Ending up with some more or some less lines of code is not an issue for me, the goal is not having less lines of code,  if xarray plotting is less clear let's keep the raw matplotlib plotting. I would like to use the mean later approach though, and I think einstats helps with that (it is basically a collection of things that have helped me do that throughout the last couple years)

**Comment by drbenvincent** [comment]
Ok. Tried harder with plotting through xarray and got it to work

**Review by OriolAbril — COMMENTED** [review]
Looks great, thanks for taking a look at the mean later approach and getting it to work.

File: myst_nbs/mixture_models/gaussian_mixture_model.myst.md [hunk 1]
```diff
@@ -6,29 +6,33 @@ jupytext:
...
+(gaussian_mixture_model)=
 # Gaussian Mixture Model
 
-
-:::{post} Sept 18, 2021
-:tags: mixture model, pymc3.Categorical, pymc3.CategoricalGibbsMetropolis, pymc3.Dirichlet, pymc3.Metropolis, pymc3.Model, pymc3.Normal, pymc3.Potential, pymc3.Uniform
-:category: intermediate
+:::{post} April, 2022
+:tags: mixture model
+:category: beginner, classification
```
[the remark this answers is not in the document]
Comment (drbenvincent) [reply myst_nbs/mixture_models/gaussian_mixture_model.myst.md]: mistake on my part. Well spotted

**Comment by drbenvincent** [comment]
Ok. Think that's done now?

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**